CMakeLists.txt

```diff
@@ -18,3 +18,7 @@ set_source_files_properties(
     ${SOURCES}
     PROPERTIES COMPILE_FLAGS "-DLFS_CONFIG=lfs_config.h"
 )
+
+if(CONFIG_LITTLEFS_FCNTL_GET_PATH)
+    target_compile_definitions(${COMPONENT_LIB} PUBLIC -DF_GETPATH=${CONFIG_LITTLEFS_FCNTL_F_GETPATH_VALUE})
```

Comment:
This should defined in header file std-libc, like newlib, but espressif's GCC doesn't contain of this one. You know customer's code will not include header file except std-libc, so adding this macro in `littlefs_api.h` is not accepted.
Customer will use this macro `F_GETPATH` instead of value directly, so I think this is OK for them. If customer need defined a customized value, they can do this in menuconfig.